requirements.in

```diff
@@ -8,4 +8,4 @@ Flask==1.1.2
 pycurl==7.43.0.6
 awscli-cwlogs>=1.4,<1.5
 notifications-python-client==6.2.1
-git+https://github.com/alphagov/emergency-alerts-utils.git@60.0.0-alpha
+git+https://github.com/alphagov/emergency-alerts-utils.git
```

Comment:
Was this changed by running `make freeze-requirements` ?

Reply:
The "frozen" requirements file has been added to the PR (see change-set for requirements.txt)